cardano-sl.cabal

```diff
@@ -118,9 +118,20 @@ library
                         Pos.WorkMode
                         Pos.WorkMode.Class
 
+                        -- Binary serialization
+                        Pos.Binary.Communication
+                        Pos.Binary.Relay
+
+                        -- Heavy delegation
+                        Pos.Delegation.Arbitrary
+                        Pos.Delegation.Types
+
   -- Explorer (TODO: move into cardano-explorer)
   exposed-modules:
                         Pos.Explorer
+
+                        -- Needed for testing
+                        Pos.Explorer.Arbitrary
```

Comment:
I had to expose this module to get visibility over the `Arbitrary` instances, which I have used in the `CborSpec`.